gitlab/utils.py

```diff
@@ -141,8 +141,10 @@ def __new__(cls, value: Union[str, int, "EncodedId"]) -> "EncodedId":
         return super().__new__(cls, value)
 
 
-def remove_none_from_dict(data: Dict[str, Any]) -> Dict[str, Any]:
-    return {k: v for k, v in data.items() if v is not None}
+def remove_none_from_dict(
+    data: Dict[str, Any], ignore_keys: Tuple[str, ...] = ()
```

Comment:
Just feels wrong to have that `ignore_keys = ()` but I guess it is okay since it is a tuple. I know it can be an issue with dicts and lists.

Reply:
I think that's more an issue for mutable defaults yes.